docker/cygnus-ngsi/Dockerfile

```diff
@@ -341,6 +340,4 @@ COPY cartodb_keys.conf ${FLUME_HOME}/conf/
 # Define the entry point
 ENTRYPOINT ["/cygnus-entrypoint.sh"]
 
-# Ports used by cygnus-ngsi
-EXPOSE ${CYGNUS_SERVICE_PORT} ${CYGNUS_API_PORT}
```

Comment:
Looking to this removal, should  `ENV CYGNUS_API_PORT "5080"` line be also removed?

Reply:
That exposes in Dockerfile does not define all ports that tipically we use. Docker-compose.yml has all ports, and we can remove `expose`

Reply:
Personally I think this change is the wrong way up, the EXPOSE should list all potential ports you can expose

```Dockerfile
# Ports used by cygnus-ngsi
EXPOSE ${CYGNUS_SERVICE_PORT} ${CYGNUS_API_PORT}  ${CYGNUS_MYSQL_SERVICE_PORT} \
  ${CYGNUS_MONGO_SERVICE_PORT} ${CYGNUS_CKAN_SERVICE_PORT} ${CYGNUS_HDFS_SERVICE_PORT} \
  ${CYGNUS_CARTO_SERVICE_PORT} ${CYGNUS_POSTGRESQL_SERVICE_PORT} \
  ${CYGNUS_ORION_SERVICE_PORT} ${CYGNUS_POSTGIS_SERVICE_PORT} \
  ${CYGNUS_ELASTICSEARCH_SERVICE_PORT} ${CYGNUS_ARCGIS_SERVICE_PORT}
```

`EXPOSE` in the Dockerfile is primarily a **documentation** mechanism - it is stating what you could expose using `-P` for example. Which ports you *actually* open and assign is up to you.

see: https://www.ctl.io/developers/blog/post/docker-networking-rules/ as an example

Reply:
By the way, is `CYGNUS_ORION_SERVICE_PORT` supposed to be the same as `CYGNUS_MYSQL_SERVICE_PORT` or should it be 5056?